src/modules/esl-share/core/esl-share-list.ts

```diff
@@ -15,11 +17,13 @@ export interface ShareButtonConfig {
   additional?: Record<string, any>;
 }
 
+/** ShareGroupConfig type definition */
 export interface ShareGroupConfig {
   name: string;
   list: string;
```

Comment:
```suggestion
  /** Name of the group. The group can be accessed with the `group:` prefix in the component configuration */
  name: string;
  /** A list of button names separated by space */
  list: string;
```

Btw: mb we need to consider array as a type for list